bootstrap-docker-install.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+
+if [ -f /etc/os-release ]; then
+    # freedesktop.org and systemd
+    . /etc/os-release
+    OS="${ID}-${VERSION_ID}"
+else
+    # We expect RHEL or Centos 7 for now. Lets set something as fallback
+    OS="$(uname -s)-$(uname -r)"
+fi
+
+echo "checking docker executable"
+if ! [ -x "$(command -v docker)" ]; then
+  case $OS in
+  rhel*)
+    extrasrepo=$(cat /etc/yum.repos.d/redhat* | grep -E '^\[.*extras.*\]$' | grep -vE 'debug|source' | tr -d '[|]')
+    sudo yum install -y --enablerepo $extrasrepo docker
+    sudo systemctl enable docker.service
```

Comment:
maybe a `systemctl daemon-reload` here would help to already be able to call `enable` and `start` for the `docker.service`